src/openms/include/OpenMS/METADATA/ProteinIdentification.h

```diff
@@ -251,6 +251,16 @@ namespace OpenMS
       AVERAGE,
       SIZE_OF_PEAKMASSTYPE
     };
+
+    /// Trypticity
```

Comment:
Would it make sense to have this neun only once?

Reply:
neun=enum

Reply:
IMHO this should go into the enzyme structures, but this is too much effort right now to support it there.